docs/usermanual/linux-installation.rst

```diff
@@ -1,13 +1,20 @@
+.. meta::
+   :description: A sparse linear algebra library with focus on exploring fine-grained parallelism on top of the AMD ROCm runtime and toolchains
+   :keywords: rocALUTION, ROCm, library, API, tool
+
+.. _linux-installation:
+
 ===================================
-Installation and Building for Linux
+Installation on Linux
 ===================================
 
+This document provides information required to install and configure rocALUTION on Linux.
+
 -------------
 Prerequisites
 -------------
 
-- A ROCm enabled platform. `ROCm Documentation <https://rocm.docs.amd.com/>`_ has more information on
-  supported GPUs, Linux distributions, and Windows SKUs. It also has information on how to install ROCm.
+A ROCm enabled platform. For information on supported GPUs, Linux distributions, ROCm installation, and Windows SKUs, refer to `ROCm Documentation <https://rocm.docs.amd.com/>`_.
 
 -----------------------------
 Installing pre-built packages
```

Comment:
"rocALUTION can be installed from `AMD ROCm repository <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/quick-start.html>`_. " The link doesn't look right.